app/models/katello/rpm.rb

```diff
@@ -0,0 +1,65 @@
+module Katello
+  class Rpm < Katello::Model
+    include Glue::Pulp::PulpContentUnit
+    include Glue::Pulp::Rpm
```

Comment:
I suppose convention. If these types of checks and guard are no longer needed we ought to just remove them wholesale.